src/mlpack/core/data/scaler_methods/minmaxscaler.hpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file minmaxscaler.hpp
+ * @author Jeffin Sam
+ *
+ * MinMaxScaler class to scale features.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_SCALE_HPP
+#define MLPACK_CORE_DATA_SCALE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
+
+/**
+ * A simple MinMax Scaler class
+ *
+ * Given an input dataset this class helps you to scale each
+ * feature to a given range.
```

Comment:
This comment is more descriptive and I as a user wouuld be able to understand what this does, but if you want, adding some equations to describe the class's functionality could still be helpful.

Reply:
Added. Not sure how good is it.

Reply:
Yeah, I think this is a lot clearer on how the functionality works.